stock_move_description_according_sale_order_line/models/sale_order_line.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (http://www.gnu.org/licenses/lgpl-3.0.html)
+from odoo import models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def write(self, values):
+        ret = super().write(values)
+        if self.move_ids.picking_id.state != 'cancel' and self.move_ids.picking_id.state != 'done':
+            for move in self:
+                move._set_description_picking()
```

Comment:
Wrong code, better this:
```suggestion
        if "name" in values:
            for move in self.filtered(lambda x: x.move_ids).move_ids:
                if move.picking_id.state in ["cancel", "done"]:
                   move._set_description_picking()
```